packages/cli/src/commands/ci/config/index.ts

```diff
@@ -0,0 +1,45 @@
+import {Command, flags as cmdFlags} from '@heroku-cli/command'
+import {ux} from '@oclif/core'
+import color from '@heroku-cli/color'
+import * as Heroku from '@heroku-cli/schema'
+import * as shellescape from 'shell-escape'
+
+import {getPipeline} from '../../../lib/ci/pipelines'
+import {getPipelineConfigVars} from '../../../lib/api'
+
+export default class CiConfig extends Command {
+  static description = 'display CI config vars'
+
+  static examples = [
+    `$ heroku ci:config --app murmuring-headland-14719 --json
+`,
+  ]
+
+  static flags = {
+    app: cmdFlags.string({char: 'a', description: 'app name'}),
+    shell: cmdFlags.string({char: 's', description: 'output config vars in shell format'}),
```

Comment:
Any v5 flags without `hasValue: true` are booleans as far as I can tell.
```suggestion
    shell: cmdFlags.boolean({char: 's', description: 'output config vars in shell format'}),
```

Reply:
Maybe add a simple test for this flag?

Reply:
That is a good tip, thank you!

Reply:
Updated. There already was a simple test for this flag. It did require an update, though.